src/view/observer/focusobserver.js

```diff
@@ -8,6 +8,10 @@
  */
 
 import DomEventObserver from './domeventobserver';
+import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+
+const setTimeout = global.window.setTimeout;
+const clearTimeout = global.window.clearTimeout;
```

Comment:
I was using `global` in a wrong way then - not for stubbing but to avoid direct access to globals. 
@pjasiun  - please look at the tests - they're checking if render was called or not. I will fix this to use sinon's fake timers.